src/raw_statement.rs

```diff
@@ -153,7 +153,7 @@ impl RawStatement {
         r
     }
 
-    #[cfg(feature = "modern_sqlite")] // 3.7.4
+    #[cfg(all(feature = "extra_check", feature = "modern_sqlite"))] // 3.7.4
```

Comment:
These are unused code depending on the feature, but the trait impl hid it. I guess the linter couldn't tell it was impossible to invoke that trait as it was effectively sealed due to referring to an inaccessible type.